apps/kube-prometheus-stack/README.md

```diff
@@ -0,0 +1 @@
+# kube-prometheus-stack
```

Comment:
please add some more details - like which kubernetes version is supported and any other important notes from the documentation which you think might be important to migrate from rancher or for operations

Reply:
Pushed an update.
WDYT?